ichub-backend/controllers/fastapi/app.py

```diff
@@ -114,4 +114,22 @@ async def twin_management_share_catalog_part_twin(catalog_part_twin_share: Catal
 
 @app.post("/twin-management/twin-aspect", response_model=TwinAspectRead, tags=["Twin Management"])
 async def twin_management_create_twin_aspect(twin_aspect_create: TwinAspectCreate) -> TwinAspectRead:
-    return twin_management_service.create_twin_aspect(twin_aspect_create)
\ No newline at end of file
+    return twin_management_service.create_twin_aspect(twin_aspect_create)
+
+@app.get("/submodel-dispatcher/{semantic_id}/{global_id}", response_model=Dict[str, Any], tags=["Submodel Dispatcher"])
+async def submodel_dispatcher_get_submodel_content(semantic_id: str, global_id: UUID, request: Request) -> TwinAspectRead:
```

Comment:
```suggestion
async def submodel_dispatcher_get_submodel_content(semantic_id: str, global_id: UUID, request: Request) -> Dict[str, Any]:
```
Do we need the response to be of type TwinAspect, or should we use the same type as the response_model to ensure consistency?

Reply:
That's a weird one :D Of course you are right. Let's say it was the error of GitHub Copilot ;)